src/components/StrongPasswordInput/styled/index.js

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import { COLOR_WARNING, COLOR_GRAY_2 } from '../../../styles/colors';
+import lighten from '../../../styles/helpers/color/lighten';
+
+const LIGHT_COLOR_WARNING = lighten(COLOR_WARNING);
+
+const StyledContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const StyledStrenghtBar = styled.div`
+    display: flex;
+    flex-direction: row;
+`;
+
+const StyledStateBar = attachThemeAttrs(styled.div)`
+    position: relative;
+    flex-grow: 1;
+    height: 2px;
+    background-color: ${COLOR_GRAY_2};
```

Comment:
let's use a theme color here, remember that all of the colors should be customized for the theme